ouroboros-network/test/Test/Ouroboros/Network/PeerSelection/PeerMetric.hs

```diff
@@ -278,7 +279,7 @@ prop_insert_peer script =
     checkResult name peer joinedAt m =
           (\peers -> counterexample (name ++ ": peer (" ++ show peer ++ ") member of "
                                           ++ show (peers, m'))
-                                    (Set.notMember peer peers))
+                                    (Set.size peers < 5 || Set.notMember peer peers))
```

Comment:
`5` seems a bit magical to me; why this fixes the issue?

Reply:
If you read the comment above you have your answer. My guess is that 1/5 is 20/100 (20%) and that's why you need to check that.